spec/acceptance/suites/ds389/nodesets/default.yml

```diff
@@ -0,0 +1,47 @@
+<%
+  if ENV['BEAKER_HYPERVISOR']
+    hypervisor = ENV['BEAKER_HYPERVISOR']
+  else
+    hypervisor = 'vagrant'
+  end
+-%>
+HOSTS:
+  ldapserver:
+    roles:
+      - ldap
+      - default
+    platform: el-8-x86_64
+    box: centos/8
+    hypervisor: <%= hypervisor %>
+  centos8:
+    roles:
+      - client
+      - sssdv2
```

Comment:
Is this role important (similarly for the el7 client)?  I don't see these roles used in the test and they are not in the oel  nodeset.